pkg/service/oidc_provider_test.go

```diff
@@ -0,0 +1,283 @@
+package service
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/DATA-DOG/go-sqlmock"
+	"github.com/google/uuid"
+	v3 "github.com/paralus/paralus/proto/types/commonpb/v3"
+	systemv3 "github.com/paralus/paralus/proto/types/systempb/v3"
+)
+
+func performOidcProviderBasicChecks(t *testing.T, provider *systemv3.OIDCProvider, uuuid string, pruuid string) {
+	if provider.GetMetadata().GetName() != "user-"+uuuid {
+		t.Error("invalid name returned")
+	}
+	if provider.GetSpec().GetProviderName() != "provider-"+pruuid {
+		t.Error("invalid provider name returned")
```

Comment:
It would help to check mapper_url , issuer_url, etc here.

Reply:
Not sure why this was marked as resolved. The idea here to check for all the values the the `oidc_provider` returns.